src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
 description: >
   Run all tests in a MATLAB project and generate test artifacts. MATLAB includes any files in your
```

Comment:
Change first sentence to?

Run MATLAB and Simulink tests and generate artifacts.

Reply:
Then, please use the text for GitHub Actions (replacing workflow with pipeline):

https://github.com/matlab-actions/run-tests#:~:text=By%20default%2C%20MATLAB%20includes%20any%20files%20in%20your%20project%20that%20have%20a%20Test%20label.%20If%20your%20workflow%20does%20not%20use%20a%20MATLAB%20project%2C%20or%20if%20it%20uses%20a%20MATLAB%20release%20before%20R2019a%2C%20then%20MATLAB%20includes%20all%20tests%20in%20the%20root%20of%20your%20repository%20and%20in%20any%20of%20its%20subfolders.